### PR TITLE
Added the program to calculate distance

### PR DESCRIPTION
**Pull request by ArnabNath1**
#### Description of Change
I have made the program to calculate the distance between two points/

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Thank you for submitting a PR! 😄 Sadly, the documentation is not up to our [**standards**](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md). Please update the file accordingly. **Let us know if you need any help here or on our [Discord server](https://the-algorithms.com/discord).** 😃

**Comment by ArnabNath1** [comment]
What kind of changes do you want? Can you please explain it so that I can make the necessary changes?

File: geometry/distance.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#include <math.h>
+#include <stdio.h>
```
Comment (Panquesito7) [inline geometry/distance.c:2]: Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>      /// for IO operations
#include <assert.h>    /// for assert
```

File: geometry/distance.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ */
+/*
+ * imported the math.h header to implement various mathematical functions.
+ * imported the stlib.h header whidh is the general standard  purpose library of
+ * C programming language.
+ */
+
+// Function to calculate distance between 2 points
+// The X and Y coordinates are the parameters of the function
+float distance(int x1, int y1, int x2, int y2)
```
Comment (Panquesito7) [inline geometry/distance.c:15]: This should include parameter documentation, what does the function return, etc..

File: geometry/distance.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#include <math.h>
+#include <stdio.h>
+/**
+ * Author Arnab Nath
```
Comment (Panquesito7) [inline geometry/distance.c:4]: Author fields should _always_ use Markdown formatting.

File: geometry/distance.c [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+ * The code is implemented in VS Code
```
Comment (Panquesito7) [inline geometry/distance.c:5]: You should explain what is the algorithm about, how it works, a Wikipedia link, space complexity, etc.. etc..

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
You can find more details on our [**contributing guidelines**](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md).

**Pull request closed without merging**